client/test-data/test-locations-scs.json

```diff
@@ -3,17 +3,20 @@
   "community": {
     "meta": {
       "name": "EdisonPaula",
-      "symbol": "SCS",
-      "icons": "QmP2fzfikh7VqTu8pvzd2G2vAd4eK7EaazXTEgqGN6AWoD",
-      "url": "https://leu.zuerich"
+      "symbol": "EDI",
+      "icons": "QmcqHLThzvpKt67NpKRy1NHtx7KduKx3EzyFB3Yk95ra8t"
```

Comment:
I guess removal of the leu.zuerich is intentional?

Reply:
yes.